rosie/federal_senate/adapter.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import numpy as np
+import pandas as pd
+from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
+from serenata_toolbox.datasets import fetch
+
+COLUMNS = {
+    'net_value': 'reimbursement_value',
+    'recipient_id': 'cnpj_cpf',
+    'recipient': 'supplier',
+}
+
+class Adapter:
+    def __init__(self, path):
+        self.path = path
+
+    @property
+    def dataset(self):
+        path = os.path.join(self.path, 'federal_senate_reimbursements.xz')
+        if not os.path.exists(path):
+            path = self.update_datasets()
+        self._dataset = pd.read_csv(path, dtype={'cnpj_cpf': np.str}, encoding = "utf-8")
+        self.prepare_dataset()
+
+        return self._dataset
+
+    def prepare_dataset(self):
+        self.drop_null_cnpj_cpf()
+        self.rename_columns()
+        self.create_columns()
+
+    def drop_null_cnpj_cpf(self):
+        self._dataset[self._dataset['cnpj_cpf'].notnull()]
+
+    def rename_columns(self):
+        columns = {v: k for k, v in COLUMNS.items()}
+        self._dataset.rename(columns=columns, inplace=True)
+
+    def create_columns(self):
+        self._dataset['document_type'] = 'simple_receipt'
```

Comment:
Can we add a comment explaining why we need this new column with the same value for every reimbursement? Specially because I think some of them are proper receipts (_nota fiscal_ in pt-BR), aren't they?